### DISCUSSION TITLE
Ventura  13.3.1 Opencore USB won't boot.

### DISCUSSION THREAD

**Discussion by zerowe-seven** [osy86/HaC-Mini#1085] [Q&A]
**Describe the bug**
I am doing a new install for it with USB installer drive, everything is default, enabled security boot.
But I always could boot into the Opencore, the boot picker show the USB device.
But if I use F10 select the USB device(shows as Opencore), it return back without boot into it.

**Expected behavior**
I third several time with different USB installer

**Environment:**
 - Running OSX version: new install
 - Target drive OSX version:  Ventura  13.3.1
 - HaC Mini installer version: v3.3.1

**Additional context**
Tried with/without security boot option. Debug mode, reset bios, but all could boot into it. Even without any logs.
Should this be an issue on 13.3.1?

**Logs**
NA.

**Comment by osy** [comment]
It is not clear from your description what the issue is. Can you please take a picture of what you are seeing as the issue? Are you trying to boot into macOS installer but do not see an option?

**Reply by zerowe-seven** [reply]
It's like a same issue with #520.

**Comment by zerowe-seven** [comment]
BTW, I found a strange workaround, which is create two USB boot stickers and insert to the NUC together.
Then the install boot could work...
